<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoomWin</class>
 <widget class="QWidget" name="ChatRoomWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	background-color : #00a884;
}

/* VERTICAL SCROLLBAR */
 QScrollBar{
	border: none;
    background: rgba(0,0,0,0);
    width: 7px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle{	
	background-color: #00a884; /* whatsapp color */
	min-height: 30px;
	border-radius: 0px;
}
QScrollBar::handle:hover{	
	background-color: #00a884;
}
QScrollBar::handle:pressed {	
	background-color: #008b6d;
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line{
	border: none;
	background-color: #00a884;
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:hover {	
	background-color: #00a884;
}
QScrollBar::sub-line:pressed {	
	background-color: #008b6d;
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line{
	border: none;
	background-color: #00a884;
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:hover {	
	background-color: #00a884;
}
QScrollBar::add-line:pressed {	
	background-color: #008b6d;
}

/* RESET ARROW */
QScrollBar::up-arrow, QScrollBar::down-arrow{
	background: none;
}
QScrollBar::add-page, QScrollBar::sub-page{
	background: none;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="3,7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QWidget" name="chatSide" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#chatSide{
	background-color : #222e35;
	image: url(:/newPrefix/Icons/zagel copy.png);
	border : 0px;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="titleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#titleBar{
border-radius : 15px;
}
QPushButton{
border-radius : 10px;
}
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0,0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/newPrefix/Icons/zagel copy.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color : rgba(255,255,255,240);</string>
           </property>
           <property name="text">
            <string>Zagel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="min">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : rgba(0,170,0,255);
}

QPushButton:pressed{
background-color : rgba(0,170,0,200);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="max">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color :rgba(240, 136, 0 ,255);
}
QPushButton:pressed{
background-color : rgba(240 ,136, 0, 200);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : rgba(240,0,0,255);
}
QPushButton:pressed{
background-color : rgb(200, 0, 0);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="controlSide" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="ChatRooms" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,14">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="controlBar" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>85</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>85</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#controlBar{
background-color : #202c33;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2,2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="profilePhotoFrame">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>20</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item alignment="Qt::AlignLeft">
                   <widget class="QPushButton" name="profilePhoto">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>70</width>
                      <height>70</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>70</width>
                      <height>70</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton#profilePhoto{
border-image : url(:/newPrefix/Icons/5a5afbe8-1228-405d-aa18-9fb0a8cade25.jpg);
border-radius : 35px;
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QPushButton" name="statusButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Stories</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color : rgba(0,0,0,0);
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="resource.qrc">
                   <normaloff>:/newPrefix/Icons/clipart1605100.png</normaloff>:/newPrefix/Icons/clipart1605100.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QPushButton" name="newChatButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Add ChatRoom</string>
                 </property>
                 <property name="statusTip">
                  <string/>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color : rgba(0,0,0,0);
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="resource.qrc">
                   <normaloff>:/newPrefix/Icons/plus-icon-transparent-5.jpg</normaloff>:/newPrefix/Icons/plus-icon-transparent-5.jpg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="chatRoomFrame">
              <property name="styleSheet">
               <string notr="true">QWidget#chatRoomFrame{
	background-color: #111b21;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="searchRoomFrame">
                 <property name="styleSheet">
                  <string notr="true">QFrame#searchRoomFrame{
background-color : #222d34;
margin-left : 10px;
margin-right : 10px;
margin-top : 7px;
margin-bottom : 7px;
border-radius : 15px;
}
QPushButton{
background-color : rgba(0,0,0,0);
}
QLineEdit{
background-color : rgba(0,0,0,0);
color : rgba(255,255,255,200);
border:None;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8">
                  <item>
                   <widget class="QPushButton" name="searchButton">
                    <property name="toolTip">
                     <string>Search</string>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="resource.qrc">
                      <normaloff>:/newPrefix/Icons/pngwing.com.png</normaloff>:/newPrefix/Icons/pngwing.com.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>25</width>
                      <height>25</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="searchLine">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="placeholderText">
                     <string>Search</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="styleSheet">
                  <string notr="true">QScrollArea{
border:none;
}
</string>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="ScrollRooms">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>368</width>
                    <height>559</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QWidget#ScrollRooms{
	background-color: #111b21;
}</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4"/>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
